### DISCUSSION TITLE
HomeAssistant Thermostat Modus

### DISCUSSION THREAD

**Discussion by lukibazuki** [OpenCCU/OpenCCU#2007] [🇩🇪 Fragen&Antworten] [answered]
In RaspberryMatic sind die einzelnen Modis Auto oder Manual. Im HomeAssistant wird der Modus "Manual" jedoch als "Heizen" angezeigt. 
Gibt es eine Möglichkeit dies anzupassen?

**Comment by Baxxy13** [comment, marked as answer]
Die Frage wäre [>>hier<<](https://github.com/danielperna84/hahomematic/discussions) besser aufgehoben.
RaspberryMatic hat keinen Einfluss darauf wie Stati in externen Anwendungen dargestellt werden.
